### PR TITLE
fixed xbutil list bug for edge

### PR DESCRIPTION
**Pull request by Himanshu-xilinx**

### CODE REVIEW

File: src/runtime_src/core/edge/tools/xbutil/xbutil.cpp [hunk 1]
```diff
@@ -427,7 +422,14 @@ int main(int argc, char *argv[])
             return -ENOENT;
         }
     }
-// TODO: how to find usable or not
+
+    if (cmd == xcldev::SCAN || cmd == xcldev::LIST) {
+        for (unsigned i = 0; i < deviceVec.size(); i++) {
+              std::cout << " [" << i << "]:" << deviceVec[i]->name();
```
[the remark this answers is not in the document]
Comment (Himanshu-xilinx) [reply src/runtime_src/core/edge/tools/xbutil/xbutil.cpp]: used for loop as it need to print index also

[the remark this answers is not in the document]
Comment (Himanshu-xilinx) [reply src/runtime_src/core/edge/tools/xbutil/xbutil.cpp]: moved size() out of loop

File: src/runtime_src/core/edge/tools/xbutil/xbutil.cpp [hunk 2]
```diff
@@ -427,7 +422,15 @@ int main(int argc, char *argv[])
             return -ENOENT;
         }
     }
-// TODO: how to find usable or not
+
+    if (cmd == xcldev::SCAN || cmd == xcldev::LIST) {
+        unsigned size = deviceVec.size();
```
Comment (stsoe) [inline src/runtime_src/core/edge/tools/xbutil/xbutil.cpp]: unsigned what?  Please specify type.

**Comment by Himanshu-xilinx** [comment]
please someone give a rebuild.

**Comment by maxzhen** [comment]
retest this please

**Pull request merged**